src/app/work-item/work-item.service.ts

```diff
@@ -57,6 +57,27 @@ export class WorkItemService {
       .catch(this.handleError);
   }
 
+ getWorkItemTypes(): Promise<any[]> {
+    let url = this.workItemTypeUrl + '.2';
+    if (process.env.ENV == 'inmemory') {
+      url = this.workItemTypeUrl;
+    }
+    return this.http
+      .get(url, { headers: this.headers })
+      .toPromise()
+      .then(response => {
+        if (process.env.ENV == 'inmemory') {
+          // Exclusively for in memory testing
+          this.workItemTypes = response.json().data as WorkItemType[];
+          this.workItemTypes = this.workItemTypes.reverse();
+        } else {
+          this.workItemTypes = response.json().data as WorkItemType[];
```

Comment:
We don't have JSON API response format for **workitemtypes** api call. So, this will fail.